docs/Userguide_quick_start.rst

```diff
@@ -10,6 +10,101 @@ cluster.
 
 .. _mila_code:
 
+Windows
+-------
+
+If you are using Windows, here is how you should go about setting up access to the cluster:
+Note: we're assuming Windows 11, but this guide should also work for Windows 10 without too much
+trouble.
+
+.. * Install the Windows Subsystem for Linux
+   
+..    We recommend using the newest version of WSL, which is available for Windows 10 and Windows 11.
+   
+..    1. Open a command prompt (cmd)
+..    2. Enter the following command to install WSL.
+
+..          .. code-block:: console
+
+..             $ wsl --install -d ubuntu
+
+..    3. Restart your computer.
+
+..    If you encounter any difficulties or want more information, please check out `the official
+..    instructions for installing the Windows Subsystem for Linux <https://learn.microsoft.com/en-us/windows/wsl/install>`_ 
+
+* Install `Visual Studio Code <https://code.visualstudio.com/>`_.
+   * Install the Remote-SSH extension for VSCode
+     This extension can be found at `this link <https://marketplace.visualstudio.com/items?itemName=ms-vscode-remote.remote-ssh>`_.
+     
+     You can also install it from within VsCode: launch the VS Code Quick Open (Ctrl+P), paste the
+     following command, and press enter:
+     ``ext install ms-vscode-remote.remote-ssh``
+
+* Install Miniconda:
+
+   We recommend using Miniconda. It's a minimal version of Anaconda without as many pre-installed
+   packages. This makes it smaller, meaning a faster download and less disk space used.
+
+   .. code-block:: console
+
+      $ # This will download the Anaconda installer.
+      $ wget https://repo.anaconda.com/miniconda/Miniconda3-latest-Linux-x86_64.sh
+      $ bash Miniconda3-latest-Linux-x86_64.sh -b
+      $ source ~/miniconda3/bin/activate
+      $ conda init bash
```

Comment:
Oops, this still assumes that the user has WSL installed (from before). I'll replace this with instructions on using the GUI installer for Windows instead.